packages/block-library/src/block/edit.native.js

```diff
@@ -100,6 +100,11 @@ export default function ReusableBlockEdit( {
 		[ ref, clientId ]
 	);
 
+	const innerBlockCount = useSelect(
+		( select ) => select( blockEditorStore ).getBlockCount( clientId ),
+		[ clientId ]
+	);
```

Comment:
Same as https://github.com/WordPress/gutenberg/pull/45819/files#r1026355761, and for the `.native.js` files as well, whenever applicable.

Reply:
Thanks, I'll do this.